static/js/chart/draw_bar.ts

```diff
@@ -282,14 +283,19 @@ export function drawStackBarChart(
   const xAxis = svg.append("g").attr("class", "x axis");
   const tempYAxis = svg.append("g");
 
+  let marginTop = MARGIN.top;
+  if (options?.title) {
+    marginTop += addChartTitle(svg, options?.title, chartWidth);
```

Comment:
Nit: remove `?` here? (& in other instances)

Reply:
done